package.json

```diff
@@ -43,5 +43,8 @@
     },
     "dependencies": {
         "@bbc/tv-lrud-spatial": "^0.0.8"
+    },
+    "volta": {
```

Comment:
what is this?

Reply:
this is the secret formula for happiness :-D